packages/smooth_app/lib/pages/preferences/user_preferences_languages_list.dart

```diff
@@ -0,0 +1,417 @@
+// Flutter Supported Languages List
+// https://api.flutter.dev/flutter/dart-ui/Locale-class.html
+
+List<String> languageNameList = <String>[
```

Comment:
You are using list of strings 2 times right? Won't it be just better to just show the supported languages in there.

Reply:
I wanted to return Tuple, I don't know if dart supports Tuple.